controllers/add_services_test.go

```diff
@@ -86,6 +88,34 @@ var _ = Describe("add_services", func() {
 		It("should not create any services", func() {
 			Expect(newServices.Items).To(HaveLen(len(initialServices.Items)))
 		})
+
+		Context("with a change to the resource labels", func() {
```

Comment:
nit: but this should read `match labels` since we are only modifying `cluster.Spec.LabelConfig.MatchLabels` right?